vpr/src/route/rr_graph.cpp

```diff
@@ -1281,6 +1278,7 @@ static void build_bidir_rr_opins(RRGraphBuilder& rr_graph_builder,
                                  const int num_directs,
                                  const t_clb_to_clb_directs* clb_to_clb_directs,
                                  const int num_seg_types) {
+    const auto& rr_graph = g_vpr_ctx.device().rr_graph;
```

Comment:
Same comments as you see in the ``build_unidir_rr_opins()`` function. I think we can start propagate the use the of RRGraphView as an argument of these functions until reaching the ``build_rr_graph()`` function (To avoid too many code changes and impact too many files)